cmd/system-probe/config/adjust_usm.go

```diff
@@ -13,7 +13,7 @@ import (
 )
 
 const (
-	maxHTTPFrag = 160
+	maxHTTPFrag = 512 // matches hard limit currently imposed in NPM driver
```

Comment:
There is a comment in `pkg/config/system_probe.go` that needs to be updated to match this.